test_cupy.sh

```diff
@@ -15,7 +15,7 @@ pytest_opts=(
 if [ $CUDNN = none ]; then
   pytest_opts+=(-m 'not cudnn and not slow')
 else
-  pytest_opts+=(-m 'not slow')
+  pytest_opts+=(-m 'not ideep and not slow')
```

Comment:
This script is for CuPy unittest.

Reply:
Yes, I want to skip ideep tests in CuPy unittest.
Oh, I should have put `not ideep` in both clauses.

Reply:
Oh, will iDeep test code be added to CuPy, too? (not only Chainer?)

Reply:
Ok, I understand...

Reply:
Fixed.